src/_data/profiles/arisafukuzaki.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "Arisa Fukuzaki",
+  "jobTitle": "Developer Relations Engineer",
+  "company": "Storyblok",
+  "bio": "DevRel Engineer @Storyblok, GirlCode ambassador, Aikido fighter, Front-End Foxed School mentor & a Japanese tech community, Lilac organizer. 💚 Gatsby, Next.js & Remix",
```

Comment:
Just checking "Front-End Foxed School" is correct and it's not "Front-End Foxes"?